trollmoves/movers.py

```diff
@@ -329,6 +329,12 @@ def copy(self):
             ssh_connection.close()
             raise
 
+        try:
+            ssh_connection.exec_command("mkdir -pv {}".format(self.destination.path))
+        except Exception as err:
```

Comment:
If possible, add some explicit errors that can happen here.

Reply:
You mean from the mkdir command?

I tried to do that, but could not figure out how to catch the output from the exec_command

Reply:
I was thinking there were some "general" exceptions the command could raise. But it seems that the output can be collected straight from the command return values (from http://docs.paramiko.org/en/2.6/api/client.html):
```python
stdin, stdout, stderr = client.exec_command('ls -l')
```
So maybe the `stderr` could be inspected. Can't now figure out how to connect the client so can't check what the `stderr` contains in different cases :smile: